lib/CFRMessageProvider.jsm

```diff
@@ -0,0 +1,95 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+"use strict";
+const BASE_ADDONS_DOWNLOAD_URL = "https://addons.mozilla.org/firefox/downloads/file";
+const AMAZON_ASSISTANT_PARAMS = {
+  existing_addons: ["abb@amazon.com", "{75c7fe97-5a90-4b54-9052-3534235eaf41}", "{ef34596e-1e43-4e84-b2ff-1e58e287e08d}", "{ea280feb-155a-492e-8016-ac96dd995f2c}", "izer@camelcamelcamel.com", "amptra@keepa.com", "pricealarm@icopron.ch", "{774f76c7-6807-481e-bf64-f9b7d5cda602}"],
+  open_urls: ["smile.amazon.com", "www.audible.com", "www.amazon.com"]
+};
+const FACEBOOK_CONTAINER_PARAMS = {
+  existing_addons: ["@contain-facebook", "{bb1b80be-e6b3-40a1-9b6e-9d4073343f0b}", "{a50d61ca-d27b-437a-8b52-5fd801a0a88b}"],
+  open_urls: ["www.facebook.com"]
+};
+
+const CFR_MESSAGES = [
+  {
+    id: "AMAZON_ASSISTANT_1",
+    template: "cfr",
+    notification_text: "Recommendation",
+    heading_text: "Recommended Extension",
+    info_icon: {
+      label: "why_seeing_this",
+      url: "https://support.mozilla.org"
```

Comment:
Out of interest will we have a specific CFR help page?

Reply:
great question, i have no idea.... @k88hudson ?

Reply:
I'm assuming yes, we'll need to request this from shell

Reply:
1. Extension recommendation (To Google Translate, Enhancer for YouTube, Wikipedia Context Menu Search, Reddit Enhancement Suite, Facebook Container): https://support.mozilla.org/1/firefox/%VERSION%/%OS%/%LOCALE%/extensionrecommendations
2. Extension promotion (Amazon Assistant):  https://support.mozilla.org/1/firefox/%VERSION%/%OS%/%LOCALE%/extensionpromotions

Reply:
So... apparently we'll need to have dynamic values :'(

Reply:
let's make this `sumo_path` and the first part of the URL can be handled by the template. Can you also update the schema when you do that?